samcli/lib/sync/flows/alias_version_sync_flow.py

```diff
@@ -107,3 +115,13 @@ def _get_resource_api_calls(self) -> List[ResourceAPICall]:
     def _equality_keys(self) -> Any:
         """Combination of function identifier and alias name can used to identify each unique SyncFlow"""
         return self._function_identifier, self._alias_name
+
+    def _get_version_alias_if_exists(self) -> Optional[str]:
+        try:
+            return str(
+                self._lambda_client.get_alias(
+                    FunctionName=self.get_physical_id(self._function_identifier), Name=self._alias_name
+                ).get("FunctionVersion")
+            )
```

Comment:
Wrapping `.get` method response with `str` will cause `None` value to be returned as `"None"` (string), so the IF statement above will not work as intended.
```suggestion
            return self._lambda_client.get_alias(
                FunctionName=self.get_physical_id(self._function_identifier), Name=self._alias_name
            ).get("FunctionVersion")
```

Reply:
Hi @driverpt, thank you for your contribution. 
We would like to circle back to the suggestion. Feel free to mention any of us here after you get a change to implement the change.

Reply:
Hello,

What do you suggest to do here? I'm not expert in Python